### PR TITLE
Added a setup script to make the OMSens installation easier

### PR DESCRIPTION
**Pull request by adeas31**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,44 +1,21 @@
...
-
-```
-cd "C:\Program Files\OpenModelica1.16.0-dev-64bit\OMSens\fortran_interface"
-gfortran -c Rutf.for Rut.for Curvif.for
-f2py.exe -c -I. Curvif.o Rutf.o Rut.o -m curvif_simplified curvif_simplified.pyf Curvif_simplified.f90 --compiler=mingw32
-"C:\Program Files\OpenModelica1.16.0-dev-64bit\bin\OMEdit.exe"
+```bash
+$ cd /path/to/OpenModelica/OMSens
+$ python setup.py install
 ```
```
[the remark this answers is not in the document]
Comment (JKRT) [reply README.md:17]: It also seems that python-numpy is needed

File: setup.py [hunk 2]
```diff
@@ -0,0 +1,115 @@
+#!/usr/bin/env python
```
Comment (JKRT) [inline setup.py:1]: Got the following error 
```

/usr/lib/python3.5/distutils/dist.py:261: UserWarning: Unknown distribution option: 'python_requires'
  warnings.warn(msg)
/usr/lib/python3.5/distutils/dist.py:261: UserWarning: Unknown distribution option: 'install_requires'
  warnings.warn(msg)
running install
running build
running install_egg_info
Writing /usr/local/lib/python3.5/dist-packages/OMSens-1.0.0.egg-info
error: [Errno 13] Permission denied: '/usr/local/lib/python3.5/dist-packages/OMSens-1.0.0.egg-info'
````

I solved this by installing numpy, maybe it was already installed for your Python3 
With sudo I got 
```
/usr/lib/python3.5/distutils/dist.py:261: UserWarning: Unknown distribution option: 'python_requires'
  warnings.warn(msg)
/usr/lib/python3.5/distutils/dist.py:261: UserWarning: Unknown distribution option: 'install_requires'
  warnings.warn(msg)
running install
running build
running install_egg_info
Writing /usr/local/lib/python3.5/dist-packages/OMSens-1.0.0.egg-info
Rut.for:760:72: Warning: Deleted feature: PAUSE statement at (1)
Rut.for:764:72: Warning: Deleted feature: PAUSE statement at (1)
Rut.for:788:72: Warning: Deleted feature: PAUSE statement at (1)
Rut.for:1073:72: Warning: Deleted feature: ASSIGN statement at (1)
Rut.for:1077:16:

 20    go to next, (30,40,70,80)
                1
Warning: Deleted feature: Assigned GOTO statement at (1)
Rut.for:1079:72: Warning: Deleted feature: ASSIGN statement at (1)
Rut.for:1084:72: Warning: Deleted feature: ASSIGN statement at (1)
Rut.for:1087:72: Warning: Deleted feature: ASSIGN statement at (1)
CURVI files compiled.
Traceback (most recent call last):
  File "setup.py", line 104, in <module>
    f2py_call = call([f2py_path, "-c", "-I.", "Curvif.o", "Rutf.o", "Rut.o", "-m", "curvif_simplified", "curvif_simplified.pyf", "Curvif_simplified.f90"], cwd="fortran_interface")
  File "/usr/lib/python3.5/subprocess.py", line 557, in call
    with Popen(*popenargs, **kwargs) as p:
  File "/usr/lib/python3.5/subprocess.py", line 947, in __init__
    restore_signals, start_new_session)
  File "/usr/lib/python3.5/subprocess.py", line 1551, in _execute_child
    raise child_exception_type(errno_num, err_msg)
FileNotFoundError: [Errno 2] No such file or directory: 'f2py'
```

Comment (adeas31) [reply setup.py:1]: `numpy` should be installed as part of `pandas`. The real problem seems to be, 

```
/usr/lib/python3.5/distutils/dist.py:261: UserWarning: Unknown distribution option: 'python_requires'
  warnings.warn(msg)
/usr/lib/python3.5/distutils/dist.py:261: UserWarning: Unknown distribution option: 'install_requires'
  warnings.warn(msg)
```

Comment (adeas31) [reply setup.py:1]: Maybe you first need todo `pip install setuptools`

Comment (JKRT) [reply setup.py:1]: Will try it out for me the script started to wrok after I installed numpy, still I got those messages for that case as well. I am still rebuilding OMC.. (I had some other unrelated compilation issues)

Comment (JKRT) [reply setup.py:1]: I will check it out tomorrow and or later this evening as well. Hopefully I will have OpenModelica up and running until then

**Review by JKRT — COMMENTED** [review]
Initial comments 

No obvious errors when running the setup script. Still, I cannot seem to get it to run on Linux. 

No obvious error message and it seems that OMS
[18_35_9.zip](https://github.com/OpenModelica/OMSens/files/5104278/18_35_9.zip)
ens is provided with info about the model etc 

bild

When invoking I get the following information:
bild

The setup script seems to exist successfully.. maybe I missed something

For me, it also seems to install stuff for Python-2 but requires Python-3.5 to run which is a bit strange.

**Comment by adeas31** [comment]
You can't install it for python 2 as the setup script has `python_requires='>=3.0'`

Can you post the setup script output and explain what exactly is the error when you run stuff (might need to look manually at python log file OR update the plugin code to https://github.com/OpenModelica/OMSens_Qt/pull/18 so that OMEdit can report the error).

**Comment by JKRT** [comment]
> 
> 
> You can't install it for python 2 as the setup script has `python_requires='>=3.0'`
> 
> Can you post the setup script output and explain what exactly is the error when you run stuff (might need to look manually at python log file OR update the plugin code to [OpenModelica/OMSens_Qt#18](https://github.com/OpenModelica/OMSens_Qt/pull/18) so that OMEdit can report the error).

Thanks. Will try again and update the plugin!

**Pull request merged**